Assets/MixedRealityToolkit/Inspectors/MixedRealityToolkitShaderGUIUtilities.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
+{
+    /// <summary>
+    /// A collection of shared functionality for MRTK shader GUIs.
+    /// </summary>
+    public static class MixedRealityToolkitShaderGUIUtilities
+    {
+        /// <summary>
+        /// GUI content styles which are common among shader GUIs.
+        /// </summary>
+        public static class Styles
+        {
+            public static readonly GUIContent DepthWriteWarning = new GUIContent("<color=yellow>Warning:</color> Depth buffer sharing is enabled for this project, but this material does not write depth. Enabling depth will improve reprojection, but may cause rendering artifacts in translucent materials.");
+            public static readonly GUIContent DepthWriteFixNowButton = new GUIContent("Fix Now", "Enables Depth Write For This Material");
+        }
+
+        /// <summary>
+        /// Checks if the project has depth buffer sharing enabled.
+        /// </summary>
+        /// <returns>True if the project has depth buffer sharing enabled, false otherwise.</returns>
+        public static bool IsDepthBufferSharingEnabled()
```

Comment:
Did you copy this from the Optimize window helper? If so, can you not just call that utils?

Reply:
I wish I could have, but there was just a settor (not a gettor) for depth buffer sharing. But, I did gleam how you accessed the project properties pre-2019 from there!